components/brave_wallet/common/eth_request_helper_unittest.cc

```diff
@@ -426,4 +428,82 @@ TEST(EthResponseHelperUnitTest, ParseSwitchEthereumChainParams) {
   EXPECT_FALSE(ParseSwitchEthereumChainParams("{\"params\": [{}]}", &chain_id));
 }
 
+TEST(EthRequestHelperUnitTest, ParseEthSignTypedDataParams) {
+  const std::string json = R"({
+    "params": [
+      "0xCD2a3d9F938E13CD947Ec05AbC7FE734Df8DD826",
+      "{
+        \"types\" :{
+          \"EIP712Domain\": [
+            { \"name\": \"name\", \"type\": \"string\" },
+            { \"name\": \"version\", \"type\": \"string\" },
+            { \"name\": \"chainId\", \"type\": \"uint256\" },
+            { \"name\": \"verifyingContract\", \"type\": \"address\" },
+          ],
+          \"Mail\": [
+              {\"name\": \"from\", \"type\": \"Person\"},
+              {\"name\": \"to\", \"type\": \"Person\"},
```

Comment:
nit: 4 spaces for `Mail` but 2 for `EIP712Domain`

Reply:
fixed in 6bbb1c4516cb09859c699807dc35b5fd17d06b54